app.js

```diff
@@ -1,50 +1,40 @@
-$(document).on("click", function(event) {
+//Initialize function when user clicks anywhere on the document
+$(document).on("click", function() {
 
     // Here we grab the text from the input box
     // var pokemon = $("#pokeMon").val();
-
-    // console.log('Pokemon: ', pokemon)
-    // Here we construct our URL
-    var queryURL = "https://pokeapi.co/api/v2/pokemon/"+ Math.floor(Math.random()*807)+"/"
-
-
-    // Write code between the dashes below to hit the queryURL with $ajax, then 
-    // YOUR CODE GOES IN THESE DASHES. DO NOT MANUALLY EDIT THE HTML ABOVE
-    // Write code between the dashes below to hit the queryURL with $ajax, then take the response data
-    // and display it in the div with an id of movie-view
+    
+var count = 1;
+
+function fetchPokemon(){
+    
+    // Here we construct our URL, and pick our pokemon randomly by its 'id number' (pokedex number). In this case, we limit our results to the first 150 pokemon (OG)
+    
+    // var queryURL = "https://pokeapi.co/api/v2/pokemon/"+ Math.floor(Math.random()*150)+"/"
+    var queryURL = "https://pokeapi.co/api/v2/pokemon/"+count+"/"
```

Comment:
what is driving this +count+?

Reply:
var count, defined outside of function (discussed already)